templates/setup/linux/docker/debian_10.sh

```diff
@@ -1,27 +1,23 @@
 #!/bin/bash
 
-# Install docker
-
+# Update and Install Dependencies
 sudo apt-get update
 
- sudo apt-get install -y \
-    ca-certificates \
-    curl \
-    gnupg \
-    lsb-release
+sudo apt-get install -y \
+  ca-certificates \
+  curl \
+  gnupg \
+  lsb-release
 
-curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+# Add Docker’s Official GPG key
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
 
+# Setup Stable repo
 echo \
-  "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian \
-  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/debian \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list >/dev/null
 
+# Install Docker
 sudo apt-get update -y
-sudo apt-get install -y docker-ce docker-ce-cli containerd.io
-
-# Install docker-compose
-
-DOCKER_CONFIG=${DOCKER_CONFIG:-$HOME/.docker}
-mkdir -p $DOCKER_CONFIG/cli-plugins
-curl -SL "https://github.com/docker/compose/releases/download/v2.6.1/docker-compose-$(uname -s)-$(uname -m)" -o $DOCKER_CONFIG/cli-plugins/docker-compose
-chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
\ No newline at end of file
+sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
```

Comment:
Is this `docker-compose` or `docker compose`? I'm not sure if the apt package is `docker compose`. We want `docker compose` as is now the correct and latest plugin. That's why we were using the above instructions to install it

Reply:
[Here](https://docs.docker.com/engine/install/debian/#install-using-the-repository), at the docker documentation, you will see that the way to install and get the plugin `docker compose` is installing `docker-compose-plugin`.